views/search.handlebars

```diff
@@ -1,26 +1,33 @@
-<h3 class="pb-3">{{title}}</h3>
+<div class="container-fluid h-100 d-flex justify-content-center">
+    <div class="col-xl-10 col-lg-10 col-md-12 col-sm-12 col-12 pt-4">
+        <div class="row">
+            <div class="col-auto pr-0">
+                <h3 class="pb-3 header-text">{{title}}</h3>
+            </div>
+            <div class="col text-left">
+                <a href="/faq" target="_blank"><i class="fas fa-info-circle text-info-bs"
+                        title="Learn about our point system"></i></a>
+            </div>
+        </div>
 
- 
-<a href="/search/pointsFAQ" class = "btn btn-info" role = "button">Learn more about the point system</a>
-<div class="card-columns"></div>
-        <br>
-        <div class="row row-cols-2">
+        <div class="card-columns"></div>
+        <div class="row">
             {{#each books}}
-                <div class="col-6 col-sm-4 col-md-3 col-lg-2 col-xl-2" id="{{id}}">
-                    <div class="card h-100 cards">
-                        <img src="{{imgUrl}}" class="card-img-top cover-art">
-                        <button type="button" class="btn btn-primary btn-details" onclick = "bookDetails({{isbn}}, {{pointcost}})">View Details</button>
-                        <button type="button" class="btn btn-success btn-request" onclick="requestBook({{id}}, {{pointcost}}, '{{title}}', {{userPoints}})">Request A Copy</button>
-                        {{!-- <div class="card-body"> --}}
-                            <p class="card-text">{{title}}</p>
-                            <p class = "card-text">{{author}}</p>
-                            {{!-- <p class = "card-text">{{pointcost}}</p>  --}}
-                        {{!-- </div> --}}
-                    </div>
+            <div id="{{id}}"
+                class="card cards border rounded bg-light p-3 col-xl-2 col-lg-3 col-md-3 col-sm-4 col-5 mr-3 mb-3">
+                <img src="{{imgUrl}}" class="card-img-top cover-art pb-1">
+                <div class="card-body px-0 pt-1 pb-1 text-center">
+                    <p class="card-text mb-0">{{title}}</p>
+                    <p class="card-text">{{author}}</p>
                 </div>
-        
+                <button class="btn btn-small small btn-info-bs mb-1">View Details</button>
```

Comment:
I noticed you removed the onclick function from this button. It was calling the google api and  setting the contents of the detail modal. This was done due to a suggestion at last Sunday's meeting.  It was dynamically setting the content of the modal as its contents changed for each book when the button was pressed. I thought you said this was ok as long I was setting the content dynamically.

Reply:
I interpreted it as needing to behave like Kylee's form when we add a book by ISBN.